src/frame.h

```diff
@@ -715,6 +715,8 @@ bool
 fget_visible(const struct frame *f);
 bool_bf
 fget_iconified(const struct frame *f);
+bool_bf
+fget_pointer_invisible(const struct frame *f);
```

Comment:
Why is it `bool` in the C file and `bool_bf` in the header?

Reply:
Changed header to bool (and make appropriate change in frames.rs as well). The documentation states:
```/* The type of bool bitfields.  Needed to compile Objective-C with
   standard GCC.  It was also needed to port to pre-C99 compilers,
   although we don't care about that any more.  */
```
So it sounds like bool_bf is legacy at this point.

Reply:
As discussed in brotzeit's PR there are reasons we need BoolBF. But there is no reason to export it from C to Rust here.